cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -517,6 +520,10 @@ private TwillPreparer prepareExploreContainer(TwillPreparer preparer) {
       }
     }
 
+    if (UserGroupInformation.isSecurityEnabled()) {
+      preparer.addSecureStore(secureStoreUpdater.update(null, null));
+    }
+
```

Comment:
This above lines are not required since `secureStoreUpdater` is already being added in `prepare`, right?